tests/mock_tables/appl_db.json

```diff
@@ -612,6 +626,15 @@
     "alias": "etp32",
     "speed": 100000
   },
+  "MGMT_PORT_TABLE:eth0": {
```

Comment:
>MGMT_PORT_TABLE [](start = 3, length = 15)

As design doc, we want to use configDB and stateDB (even front panel, we will move there in near future). Please also test oper_status and admin_status.